File: src/main/java/uk/co/threebugs/mochiwhattotrade3/accountsetupgroups/AccountSetupGroupsController.java
package uk.co.threebugs.mochiwhattotrade3.accountsetupgroups;

import lombok.RequiredArgsConstructor;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.List;

@RequiredArgsConstructor
@RestController
public class AccountSetupGroupsController {

    private final AccountSetupGroupsService accountSetupGroupsService;

    @GetMapping("/accountSetupGroups")
    public List<AccountSetupGroupsDto> findAll() {
        return accountSetupGroupsService.findAll();
    }
}

File: src/main/kotlin/uk/co/threebugs/mochiwhattotrade3/accountsetupgroups/AccountSetupGroupsController.kt
package uk.co.threebugs.mochiwhattotrade3.accountsetupgroups

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class AccountSetupGroupsController(private val accountSetupGroupsService: AccountSetupGroupsService) {
    @GetMapping("/accountSetupGroups")
    fun findAll(): List<AccountSetupGroupsDto> {
        return accountSetupGroupsService.findAll()
    }
}
